lib/rack/ecg.rb

```diff
@@ -52,10 +52,10 @@ def call(env)
     def build_check_classes(check_names)
       check_names = Array(check_names) # handle nil, or not a list
       check_names = check_names | DEFAULT_CHECKS # add the :http check if it's not there
-      check_names.map{|check_name|
+      check_names.map { |check_name, check_params|
         check_class = CheckRegistry.instance[check_name]
         raise "Don't know about check #{check_name}" unless check_class
-        check_class
+        [check_class, check_params]
```

Comment:
@petervandoros Yeah, I can, although the existing spec already exercises this pathway anyway